### DISCUSSION TITLE
Enable moving, scaling and rotating nodes after placing them

### DISCUSSION THREAD

**Discussion by ortizen** [CariusLars/ar_flutter_plugin#49] [Q&A]
Hello @CariusLars 
Thank you so much for this amazing package! I am just looking to implement a feature in a Flutter App to make rough distance estimations and I am thinking of using AR to perform the measures using world location in anchors. But to do that I would need to move the anchors after placing them. I saw this is an enhancement in your road map and wonder if you are still planning to release this functionality in the near future?
Thank you again for all the hard work on this and let me know if I can help, though not much of an ARCore or ARKit programmer (yet)   :=(

**Comment by CariusLars** [comment]
Hi @ortizen, thanks for your message! The feature is definitely still on my road map, I'll be able to give you an update for an estimated release date in 1-2 weeks (should be quite soon hopefully)!

**Comment by CariusLars** [comment]
Hey @ortizen ,
sorry for the late comeback on this! In the new version of the plugin (v0.6.0), rotating and moving the nodes using gestures is enabled :) 
Scaling doesn't work yet, but maybe that's something you could look into? should work pretty similarly to how panning and rotating is implemented.
Best,
Lars

**Reply by ortizen** [reply]
Hello @CariusLars
What a great news! Really appreciate you improvements in an already amazing package. Let me review the new version and tack the scaling feature ASAP in a PR
Thank you again man!

**Reply by CariusLars** [reply]
great, looking forward to your PR!
